sys-apps/sed/files/sed-4.8-avoid-noreturn-diagnostic.patch

```diff
@@ -0,0 +1,19 @@
+From 0cc39712803ade7b2d4b89c36b143dad72404063 Sun Oct 18 00:00:00 2020
```

Comment:
Can you include a link to the relevant upstream commit as well?

Reply:
Thanks, done.